class LoginError(Exception):
    pass


class UsernameError(Exception):
    pass
